homeassistant/components/elevenlabs/quality_scale.yaml

```diff
@@ -0,0 +1,85 @@
+rules:
+  # Bronze
+  action-setup:
+    status: done
+    comment: >
+      Only entity services
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: todo
+  entity-event-setup:
+    status: exempt
+    comment: |
+      Entities of this integration does not explicitly subscribe to events.
+  entity-unique-id: done
+  has-entity-name: todo
+  runtime-data: done
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry: todo
+  # Silver
+  config-entry-unloading: todo
+  log-when-unavailable: todo
+  entity-unavailable: todo
```

Comment:
I am not sure if this is possible, since the TTS platform does not have state, and I don't think we can poll them every x minutes

Reply:
Fair. I was already planning ahead a bit I think, as I am currently thinking about adding sensors to this integration, in which case this would be a todo. For now, this should be exempt though